tenders/hvt/hvt_dumpcore_openbsd_x86_64.c

```diff
@@ -21,21 +21,96 @@
 /*
  * hvt_dumpcore_openbsd_x86_64.c: Glue between the dumpcore module and OpenBSD
  * (x86_64).
+ *
+ * Please note the core generated on OpenBSD is not readable by gdb on OpenBSD.
  */
 
+#include <sys/ioctl.h>
+#include <machine/vmmvar.h>
+
+#include "hvt.h"
+#include "hvt_openbsd.h"
+#include "hvt_cpu_x86_64.h"
+
 int hvt_dumpcore_supported()
 {
-    return -1;
+    return sizeof (prstatus_t);
```

Comment:
This should be `return 0;`. I agree that the semantics are not obvious.

Reply:
I will fix this up along with any other changes required.